scripts/build.sh

```diff
@@ -9,16 +9,29 @@ set -ev
 python3 -m pip install --upgrade pip setuptools wheel
 
 BASEDIR=$(dirname $(readlink -f $(dirname $0)))
+DISTDIR=dist
 
 (
   cd $BASEDIR
-  mkdir -p dist
-  rm -rf dist/*
+  mkdir -p $DISTDIR
+  rm -rf $DISTDIR/*
```

Comment:
looks like this already existed, but: will we ever need a "-f" here? I guess if someone didn't something wrong, I'd feel more comfortable having them fix manually. But maybe there's the possibility of non-write directories here.

Reply:
As long as we're the only ones writing to the dist dir I don't think we need `-f`, but I assume we just want to wipe out the entire dir with each build.

Reply:
I guess I worry about some freak accident that destroys user's directories without warning. but it's highly unlikely either way.